src/modules/UI/components/ControlPanel/style.js

```diff
@@ -183,5 +182,9 @@ export default {
   },
   underlay: {
     color: `${THEME.COLORS.PRIMARY}${THEME.ALPHA.LOW}`
+  },
+  debug: {
```

Comment:
I'll approve this pull request but it appears that this `debug` style is not being used.